.github/workflows/codeql-cosmos.yml

```diff
@@ -0,0 +1,40 @@
+name: "CodeQL for cosmos"
```

Comment:
same here, we are going to run code analysers on our own infra!

Reply:
That does not provide the public with its result